modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud/remove_user_from_group_command.rb

```diff
@@ -62,7 +62,8 @@ def call(user:, group: @group)
         when "104"
           Util.error(:error, "Insufficient privileges")
         when "105"
-          Util.error(:error, "Failed to remove user from group")
+          message = Nokogiri::XML(response.body).xpath('/ocs/meta/message').text
+          Util.error(:error, "Failed to remove user from group: #{message}")
```

Comment:
🟢 WDYT of interpolating the group and user into the message, in order to make it clearer to the end user?